app/views/layouts/base.html.erb

```diff
@@ -69,14 +70,21 @@
         <% end %>
       <% end %>
     </div>
+  </div>
+  <main class="site-wrap bg-light-blue" id="main-content">
```

Comment:
Were you able to find if `site-wrap` has any effect? I wondered if it's actually doing anything for us, or if we could just remove it.

Reply:
We need the flex so that the footer is on the very bottom of the page. I don't think we need any other declarations.

Reply:
> We need the flex so that the footer is on the very bottom of the page. I don't think we need any other declarations.

I wondered about that as well, but is that being handled by the `site` class on the `<body>` tag?

Reply:
I have tried messing around with this as well as some other variations to get that footer to stick, but I haven't found a way to do that unless I am missing something.

Reply:
> I have tried messing around with this as well as some other variations to get that footer to stick, but I haven't found a way to do that unless I am missing something.

Ah, I just tested and I see what you mean. I guess we rely on the flex styles to let the main area grow to push the footer to the bottom.

Disregard this then, sorry for the misunderstanding 👍